pose_diff/core/run.py

```diff
@@ -35,6 +35,8 @@ def __init__(self,trainer_npy,user_npy, video_name, exercise=2,diffing='increase
 
             cutframes.append(user)
             for point, angle in zip(point_gap, angle_gap):
+                print('point %d' % point)
+                print('angle %d' % angle)
```

Comment:
When try to give some errors to standard user and trainer , they made nan value here

Reply:
It's also probably occurs framecut. But it was resolved. Now, this function will run perfectly.

Reply:
Thank you